### DISCUSSION TITLE
IRsend does not send cmd on URC-150

### DISCUSSION THREAD

**Discussion by axi92** [arendst/Tasmota#16443] [Q&A] [answered]
I have 2 URC-150 all with the newest Tasmota-ir 12.1.1(ir)
I recived this message
```
{"Protocol":"FUJITSU_AC","Bits":128,"Data":"0x1463001010FE0930A10003000000200C","Repeat":0,"IRHVAC":{"Vendor":"FUJITSU_AC","Model":1,"Mode":"Auto","Power":"On","Celsius":"On","Temp":26,"FanSpeed":"Low","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}
```
from my Remote.
But when I send it on the console the other does not recive it. So I guess it is not sent. Also I dont see any lamps on my phone camera.

To test if it is sending at all I just grabed one random cmd from a github [issue](https://github.com/arendst/Tasmota/issues/5523) ({"Protocol":"RC6","Bits":36,"Data":0x8052100C}) that is working with sending and receiving by the other one. Also showing on my phone camera.

For the config I used this template: `{"NAME":"NX-4519-675","GPIO":[0,0,0,0,52,51,0,0,0,0,8,0,0],"FLAG":0,"BASE":18}`
from https://templates.blakadder.com/auvisio_S06.html

Any Idea why the read command is not beeing sent by my device? Is my template config right?

**Comment by s-hadinger** [comment, marked as answer]
What command did you try? It should be something like the following as described in the doc:

`IRhvac {"Vendor":"FUJITSU_AC","Model":1,"Mode":"Auto","Power":"On","Celsius":"On","Temp":26,"FanSpeed":"Low","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}`

**Reply by axi92** [reply]
How did you came to that conclusion? I thought I just copy the received cmd and paste it as IRsend?

**Reply by s-hadinger** [reply]
You're not answering my question: what command did you try?

First HVAC commands require the command `IRhvac` and not `IRsend`. Second you just paste the content after `IRHVAC`.

**Reply by axi92** [reply]
Ah ok ty for your help now it makes sense =)
And it works now!

I tried this:
`IRsend {"Protocol":"FUJITSU_AC","Bits":56,"Data":"0x146300101002FD","Repeat":0,"IRHVAC":{"Vendor":"FUJITSU_AC","Model":1,"Mode":"Auto","Power":"Off","Celsius":"On","Temp":16,"FanSpeed":"Auto","SwingV":"Off","SwingH":"Off","Quiet":"Off","Turbo":"Off","Econo":"Off","Light":"Off","Filter":"Off","Clean":"Off","Beep":"Off","Sleep":-1}}`
